app/code/Magento/Customer/Api/AccountManagementInterface.php

```diff
@@ -31,13 +31,15 @@ interface AccountManagementInterface
      * @param \Magento\Customer\Api\Data\CustomerInterface $customer
      * @param string $password
      * @param string $redirectUrl
+     * @param array $extensions
      * @return \Magento\Customer\Api\Data\CustomerInterface
      * @throws \Magento\Framework\Exception\LocalizedException
      */
     public function createAccount(
         \Magento\Customer\Api\Data\CustomerInterface $customer,
         $password = null,
-        $redirectUrl = ''
+        $redirectUrl = '',
+        $extensions = []
```

Comment:
Unfortunately we can't change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/), especially on the API interfaces, we can't modify them at all.